css/style.css

```diff
@@ -2018,3 +2018,84 @@ the home page, intro pitch and your-code-here layed out vertically */
 dt.d_decl:hover .decl_anchor {
     visibility: visible;
 }
+
+/* Friends of D */
+.orgs-using-d {
+    width: 100%;
+    padding-left: 20px;
+}
+
+.orgs-using-d-cell {
+    margin-left: auto;
+    margin-right: auto;
+    width: 250px;
+    display: inline-block;
+    vertical-align: top;
+    margin-bottom: 0.5em;
+    padding-left: 25px;
+    padding-right: 25px;
+}
+
+.orgs-using-d-cell > a , .frontpage-orgs > a {
+    text-decoration: none;
+}
+
+.orgs-using-d-img-wrapper {
+    width: 250px;
+    height: 100px;
+    padding-bottom: 10px;
+    text-align: center;
+}
+
+.orgs-using-d img {
+    max-width: 200px;
+    max-height: 100px;
+}
+
+.orgs-using-d h3 , .frontpage-orgs h3 {
+   margin: 0.4em;
+   margin-left: 0;
+   margin-right: 0;
+   color: black;
+   text-align: center;
+}
+
+.orgs-using-d img , .frontpage-orgs img {
+    /* vertical centering in CSS - cross-platform way */
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
```

Comment:
`transform` is relatively new. Maybe use one of the older tricks, like `line-height: 100px` on the parent, and `vertical-align: middle` here.